frontend/tests/Utils.php

```diff
@@ -97,12 +97,38 @@ public static function gradeRun(
             "\x1f\x8b\x08\x08\xaa\x31\x34\x5c\x00\x03\x66\x6f" .
             "\x6f\x00\x03\x00\x00\x00\x00\x00\x00\x00\x00\x00"
         );
-        // An empty zip file.
+        // Creating the zip file.
+        $outputFilesContent = [
+            'easy.00.out' => '3',
+            'easy.01.out' => '5',
+            'medium.00.out' => '300',
+            'medium.01.out' => '6912',
+            'sample.out' => '3',
+        ];
+
+        $zipFile = tmpfile();
+        $zipPath = tempnam(sys_get_temp_dir(), 'files');
```

Comment:
```suggestion
        $zipPath = stream_get_meta_data($zipFile)['uri'];
```

hay que evitar el uso de `tempnam()`, porque ese no se limpia automáticamente y nada más va a dejar basura en `/tmp`.

Reply:
Listo.